success-criteria.md

```diff
@@ -443,39 +425,29 @@ stakeholder group.
 -   The publisher ad-funded business model is supported by addressing marketers
     needs and wants.
 
-    -   [Impacting these marketer interests, reduces publishers' revenue earned](https://services.google.com/fh/files/misc/disabling_third-party_cookies_publisher_revenue.pdf)
+    -   The capacity to positively match type and quality of publishers' content with advertising content
 
-    -   Negatively impacts the quantity and quality of publishers’ content
+    -   Supporting quantity and quality of publisher content to create a diverse and competitive field of ad opportunities, which supports diverse publishers.
 
-    -   Reducing quantity and quality of publishers’ content, conflicts with the
-        interests of an open society
-
--   Marketers who invest in cross-publisher advertising need scaled,
-    interoperable measurement and control:
+-   Marketers who invest in cross-publisher advertising need reporting, insights and recommendations to improve future business outcomes:
 
     -   Pre-campaign media planning and forecasting
 
-        -   Data enrichment services (e.g., context, geographic, technographic,
-            audience)
+        -   Services which provide additional dimensions to target, including general demographic definitions.
 
     -   Intra-campaign delivery
 
-        -   Real-time feedback to improve content matching and cross-publisher
-            budget reallocation to better engage with prospects and customers
-            based upon advertising return on investment (ROI)
+        -   Feedback that can be used within less than the scope of time of a running campaign to measure and react to inventory changes across multiple ad-presenting-surfaces in a way that allows them to reasonably maximize their return on investment (ROI)
 
         -   Fraud and robot detection
 
     -   Post-campaign delivery
 
-        -   Independent verification of delivery and measurement
+        -   Independently verifiable measurement of delivery of ads
 
         -   Attribution of first-party engagement to prior third-party exposure
 
-        -   Aggregate content consumption trends
-
-        -   Reporting, insights and recommendations to improve future business
-            outcomes
+        -   Aggregate trend data describing the general contexts in which ads were seen and the reactions users had to those ads in order to refine targeting against specific contexts
```

Comment:
Non-specific. What, in this context, is content consumption? In the previous uses of the phrase in this document it seems like you are implying 'behavioral' trends here. I think it would be better to be more specific about the general, non-technical, requirements and leave the question of the technical fulfillment of those requirements to other documents.

How about this instead?